hack/lib/constants.sh

```diff
@@ -10,11 +10,11 @@ readonly OS_REQUIRED_GO_VERSION="go${OS_BUILD_ENV_GOLANG}"
 readonly OS_GLIDE_MINOR_VERSION="13"
 readonly OS_REQUIRED_GLIDE_VERSION="0.$OS_GLIDE_MINOR_VERSION"
 
-readonly OS_GOFLAGS_TAGS="include_gcs include_oss containers_image_openpgp"
-readonly OS_GOFLAGS_TAGS_LINUX_AMD64="gssapi"
-readonly OS_GOFLAGS_TAGS_LINUX_S390X="gssapi"
-readonly OS_GOFLAGS_TAGS_LINUX_ARM64="gssapi"
-readonly OS_GOFLAGS_TAGS_LINUX_PPC64LE="gssapi"
+readonly OS_GOFLAGS_TAGS="include_gcs include_oss containers_image_docker_daemon_stub containers_image_openpgp containers_image_ostree_stub exclude_graphdriver_btrfs exclude_graphdriver_devicemapper exclude_graphdriver_zfs"
+readonly OS_GOFLAGS_TAGS_LINUX_AMD64="gssapi seccomp selinux"
```

Comment:
What?  No.

Reply:
We can do without these so long as we stick to using chroot for isolation, dropping.